frontend/public/metalkube/components/host/menu-actions.ts

```diff
@@ -0,0 +1,25 @@
+import { Kebab } from '../utils/okdutils';
+
+const menuActionDrainHost = (kind, host) => ({
+  hidden: false, // TODO(jtomasek): use canDrainHost selector
+  label: 'Drain Host',
+  callback: () => {
```

Comment:
@mareklibra @rawagner  is it okay to have actions in the UI which don't do anything?

Reply:
@suomiy Im not in favour, but If they need it for the demo..

Reply:
Considering context of the effort, I would not block on this.

Reply:
ok